From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 09:26:13 +0000
Subject: Translated using Weblate (German)

Currently translated at 100.0% (3 of 3 strings)

Translation: openkylin-huanghe/kylin-os-manager
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager/de/
---
 src/application/translations/kylin-os-manager_de.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/application/translations/kylin-os-manager_de.ts b/src/application/translations/kylin-os-manager_de.ts
index 42afd1e..fe76a50 100644
--- a/src/application/translations/kylin-os-manager_de.ts
+++ b/src/application/translations/kylin-os-manager_de.ts
@@ -13,7 +13,7 @@
         <location filename="../mainwindow.cpp" line="219"/>
         <location filename="../mainwindow.cpp" line="399"/>
         <source>Kylin Os Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>Kylin Manager</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="400"/>
